spec/fixtures/poro.rb

```diff
@@ -165,6 +227,24 @@ class SerializableEmployee < JSONAPI::Serializable::Resource
 
     attribute :first_name
     attribute :last_name
+    attribute :age
+
+    is_admin = proc { |c| @context && @context.current_user == 'admin' }
+    attribute :salary, if: is_admin do
+      100_000
+    end
+
+    extra_attribute :stack_ranking do
+      rand(999)
+    end
+
+    extra_attribute :admin_stack_ranking, if: is_admin do
```

Comment:
Would like to be able to do `def admin?` and provide `if: :admin?` as my guard argument as well.

Reply:
agreed. just getting the current API passing right here but for sure we need to do that,